### PR TITLE
adding pharmacy data quality to docs

### PR DESCRIPTION
**Pull request by bradmontierth**

### CODE REVIEW

File: docs/data-quality/atomic-level.md [hunk 1]
```diff
@@ -245,18 +245,148 @@ from data_quality.primary_key_check
 
 ### Patient ID
 
+Similar to the medical_claim table, the patient is at the center of the vast majority of the analyses we're interested in.  Therefore it's important that we check a few things related to the `patient_id` field on every pharmacy claim, specifically:
+
+1. Does every line on each claim have a value for `patient_id` populated?
+2. Is there more than 1 value for `patient_id` within a single claim (there shouldn't be)?
```
Comment (tom-tuva) [inline docs/data-quality/atomic-level.md:251]: Switch 1 to one and I would remove the part in the parenthesis

File: docs/data-quality/atomic-level.md [hunk 2]
```diff
@@ -245,18 +245,148 @@ from data_quality.primary_key_check
...
+3. Does the `patient_id` value indicated on the claim have corresponding valid eligibility during the time period when claim was rendered?
+
+The `pharmacy_claim_patient_id` table verifies whether any of these data quality problems occur in the `pharmacy_claim` table.  You can query it as follows:
+
+```sql
+select *
+from data_quality.pharmacy_claim_patient_id
+```
+
+This query returns the number of unique claim IDs that have each of these data quality problems.
```
Comment (tom-tuva) [inline docs/data-quality/atomic-level.md:261]: This query returns a table with one row per check and the count of unique claim IDs that have that particular data quality issue

File: docs/data-quality/atomic-level.md [hunk 3]
```diff
@@ -245,18 +245,148 @@ from data_quality.primary_key_check
...
+
+![Medical Claim Patient ID](/img/data_quality_medical_claim_patient_id.jpg)
```
Comment (tom-tuva) [inline docs/data-quality/atomic-level.md:263]: Need to update name and filepath

File: docs/data-quality/atomic-level.md [hunk 4]
```diff
@@ -245,18 +245,148 @@ from data_quality.primary_key_check
...
 ### Trended Claim Volume and Dollars
 
+Often we see anomalies where claim volume or dollars can change unexpectedly over time, possibly indicating a data quality problem.  Therefore we need to check whether this is the case.
+
+```sql
+select *
+from data_quality.trended_pharmacy_claim_volume_and_dollars
+```
+
+![Medical Claim Volume and Dollars](/img/data_quality_medical_claim_volume_and_dollars.jpg)
```
Comment (tom-tuva) [inline docs/data-quality/atomic-level.md:371]: Update to pharmacy

File: docs/data-quality/atomic-level.md [hunk 5]
```diff
@@ -245,18 +245,148 @@ from data_quality.primary_key_check
...
+
+In the example above we clearly have medical claims in the month of March 2021 but we have no paid amounts.
```
Comment (tom-tuva) [inline docs/data-quality/atomic-level.md:373]: Can we just add dollars to the claim check we already did above in the date trended view?

File: static/img/data_quality_pharmacy_claim_npi.jpg
Comment (tom-tuva) [inline static/img/data_quality_pharmacy_claim_npi.jpg:1]: change to result_count

File: static/img/data_quality_pharmacy_ndc.jpg
Comment (tom-tuva) [inline static/img/data_quality_pharmacy_ndc.jpg:1]: change to result_count

File: static/img/data_quality_pharmacy_prescription_details.jpg
Comment (tom-tuva) [inline static/img/data_quality_pharmacy_prescription_details.jpg:1]: change to result count

**Review by tom-tuva — COMMENTED** [review]
Looks good overall, just a few updates to make!

File: docs/data-quality/atomic-level.md (hunk 1 above)
Comment (bradmontierth) [reply docs/data-quality/atomic-level.md:251]: done

File: docs/data-quality/atomic-level.md (hunk 2 above)
Comment (bradmontierth) [reply docs/data-quality/atomic-level.md:261]: done

File: docs/data-quality/atomic-level.md (hunk 3 above)
Comment (bradmontierth) [reply docs/data-quality/atomic-level.md:263]: it is the same table for both sections

File: docs/data-quality/atomic-level.md (hunk 4 above)
Comment (bradmontierth) [reply docs/data-quality/atomic-level.md:371]: done

File: docs/data-quality/atomic-level.md (hunk 5 above)
Comment (bradmontierth) [reply docs/data-quality/atomic-level.md:373]: i think its fine to keep them as separate for the separate focus they have. one is on dates and if they are populated. the other is on volume vs dollars.

File: docs/data-quality/atomic-level.md (hunk 3 above)
Comment (bradmontierth) [reply docs/data-quality/atomic-level.md]: updated

File: static/img/data_quality_pharmacy_claim_npi.jpg
Comment (bradmontierth) [reply static/img/data_quality_pharmacy_claim_npi.jpg:1]: done

File: static/img/data_quality_pharmacy_ndc.jpg
Comment (bradmontierth) [reply static/img/data_quality_pharmacy_ndc.jpg:1]: done

File: static/img/data_quality_pharmacy_prescription_details.jpg
Comment (bradmontierth) [reply static/img/data_quality_pharmacy_prescription_details.jpg:1]: done

**Pull request merged**